src/components/Footer.js

```diff
@@ -7,10 +7,12 @@ import Image from 'react-bootstrap/Image';
 class Footer extends Component{
 	render(){
 		return (
-			<Row>
-				 <Col xs={12} sm={12} md={12} lg={12} xl={12}>
-					<div id="footer">
-						<div id="footer_social_links_container">
+				<footer id="footer">
+					<Row>
+						<Col xs={12} sm={12} md={6} className="text-center text-md-start order-last order-md-first mt-5px" id="copyright_links">
+							© 2021 xSurge. All rights reserved.
+						</Col>
+						<Col xs={12} sm={12} md={6} className="text-center text-md-end" id="footer_social_links_container">
```

Comment:
Explanation of the logic: So it centers on mobile first. But on medium screens, it's right aligned (text-md-end).